### PR TITLE
Types of quarantine table and defaults of view for depth_m and clear_hight

### PR DESCRIPTION
**Pull request by signedav**
Change the types regarding request https://github.com/QGEP/QGEP/issues/455

Effects on `qgep_import.manhole_quarantine.<inlet/outlet>_<n>_clear_hight`:
`qgep_od.reach.clear_hight` is set. Since that is `integer`, I see no problem in changing q`gep_import.vw_manhole.clear_hight` to `integer`.
 
Effects on `qgep_import.manhole_quarantine.<inlet/outlet>_<n>_depth_m`:
Is subtracted from `qgep_import.vw_manhole.co_level` what actually is `qgep_od.cover.level` and sets in the end `qgep_od.reach_point.level`. Since these table values are both `numeric(7,3)` I see no problem in changing it.

### CODE REVIEW

File: 13_import.sql [hunk 1]
```diff
@@ -25,26 +25,26 @@ CREATE TABLE qgep_import.manhole_quarantine
   photo1 text,
   photo2 text,
   inlet_3_material smallint,
-  inlet_3_clear_hight numeric(6,3),
-  inlet_3_depth_m numeric(6,3),
+  inlet_3_clear_hight integer,
```
Comment (3nids) [inline 13_import.sql]: clear_height?

Comment (3nids) [reply 13_import.sql]: fixing the column name would require view drop / create (no replace possible)

**Comment by varrieta** [comment]
Why did it fail?

**Comment by signedav** [comment]
Ha! 
Spaces in the end of a line have been the issue. :slightly_smiling_face:  At a completely different position than pointed...
@3nids Okay for you to merge that?

**Comment by signedav** [comment]
Thanks @3nids and thanks @varrieta for remembering.

**Pull request merged**